packages/code-du-travail-data/.env

```diff
@@ -0,0 +1,3 @@
+PYTHONPATH=.
+ES_HOST=elasticsearch
+ES_PORT=9200
```

Comment:
I'm sorry,  I thought these files was unused anymore since we handle the env globally in `docker-compose.yml`

Reply:
In which case we need it ?

Reply:
indeed, they are overriden by the root `docker-compose.yml`, we shouldnt need this except for explicitness maybe. will remove that!